mkdocs.yaml

```
@@ -0,0 +1,53 @@
+  - Changelog: changelog.md
+
+plugins:
+  - awesome-pages                     # simplifies configuring page titles and their order
+  - search                            # necessary for search functionality to work
+  - git-authors                       # adds authors to pages using git history
+  - include-markdown                  # allows for including Markdown files into another Markdown file
+  
+theme: 
+  name: material
+  features:
+  - announce.dismiss
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)